modules/apps/frontend-icons/frontend-icons-clay/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"dependencies": {
+		"@clayui/css": "3.32.1"
+	},
+	"main": "images/icons.svg",
```

Comment:
Also, how do you plan to load these icons? Are you going to `import` them from JavaScript? If so, who is going to take care of that? I guess the bundler should wrap them as it does for `.css` files... Or am I missing the point totally? :sweat_smile: